utils/network.go

```diff
@@ -66,22 +66,33 @@ func (n *Network) UnmarshalText(text []byte) error {
 	return n.Set(string(text))
 }
 
-func (n Network) URL() string {
+// BaseURL returns the base URL without endpoint
+func (n Network) BaseURL() string {
```

Comment:
since it's used only internally let's unexport it (i.e. `BaseURL()` -> `baseURL()`)